fpsd/attack.yaml

```diff
@@ -0,0 +1,90 @@
+#################
+# Train/Test    #
+#################
+
+world: 
+  type: 'open'  # 'closed' or 'open'
+  observed_fraction: 0.20  # fraction of the open world that can be measured by the adversary 
```

Comment:
Yep we can make this take a list at a later point, updated #64 issue to keep this suggestion